query/language.peg

```diff
@@ -105,7 +105,7 @@ expression_function  <-
     p.addStringLiteral(unescapeLiteral(buffer[begin:end]))
   }
   PAREN_OPEN
-    expressionList (__ groupByClause)?
+    expressionList { p.addGroupBy() } (__ groupByClause)?
```

Comment:
So are `group by`s meant to apply over all expressions within an expressionList?

`x, y group by z` == `x group by z, y group by z`?